src/components/ha-color-picker.ts

```diff
@@ -33,12 +33,12 @@ export class HaColorPicker extends LitElement {
 
   @property({ type: Boolean }) public disabled = false;
 
-  @query("ha-select") private _select!: HaSelect;
+  @query("ha-select") private _select?: HaSelect;
```

Comment:
How does just a typing change fix this? 🤔

Reply:
It's not just typing, layoutOptions will be not called in connected callback if the select is not yet rendered.